src/main/scala/Chisel/testers/BasicTester.scala

```diff
@@ -19,10 +22,13 @@ class BasicTester extends Module {
     * reset). If your definition of reset is not the encapsulating Module's
     * reset, you will need to gate this externally.
     */
-  def stop() {
-    when (!reset) {
-      pushCommand(Stop(Node(clock), 0))
-    }
+  def stop(): Unit = macro StopTransform.stop
```

Comment:
Gone, along with AssertTransform, PrintTransform, and WhenTransform.otherwise, all of which return Unit